packages/data/src/redux-store/metadata/selectors.js

```diff
@@ -131,3 +131,18 @@ export function isResolving( state, selectorName, args ) {
 export function getCachedResolvers( state ) {
 	return state;
 }
+
+/**
+ * Whether the store has any currently resolving selectors.
+ *
+ * @param {State} state Data state.
+ *
+ * @return {boolean} True if one or more selectors are resolving, false otherwise.
+ */
+export function hasResolvingSelectors( state ) {
```

Comment:
Depending how confident we are with this selector (it seems fine to me), we could consider making it private to start with.

Reply:
To me, one of the positive sides of metadata selectors is that they are always publicly available. That being said, I wonder if you think there's a particular need for this specific one to be private.

Reply:
Not for me. I think it's fine for this one to be public. I was suggesting it in case we missed confidence.